### PR TITLE
Update installer to v0.8.0 and python3

### PR DESCRIPTION
**Pull request by banasiakadam60**
Fixes: #1246

### CODE REVIEW

File: Installer/Installer_Win/install_script.iss [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #define MyAppExeName "golemapp.exe"
 ; NOTE: if compilation failed, make sure that this variable are set properly and golem is installed from wheel
 ; NOTE 2: make sure that you've got in {#Repository}\Installer\Inetaller_Win\deps:
-; https://www.microsoft.com/en-us/download/details.aspx?id=40784 vcredist_x86.exe
```
Comment (paulperegud) [inline Installer/Installer_Win/install_script.iss:10]: Is it vc_redist.x86.exe or vcredist_x86.exe? This looks strange.

Comment (banasiakadam60) [reply Installer/Installer_Win/install_script.iss:10]: For python 2.7 `vcredist_x86.exe`
For python 3.5 `vc_redist.x86.exe`
Powered by MS 

I'll change that in the next commit, but here it's just a comment. In the code (lines 66 and 78 it is changed)

**Comment by banasiakadam60** [comment]
#1246

**Comment by banasiakadam60** [comment]
#1247

**Pull request merged**